pyproject.toml

```diff
@@ -14,7 +14,7 @@ boto3 = "^1.16.58"
 python-dotenv = "^0.15.0"
 SQLAlchemy = "^1.3.22"
 psycopg2-binary = "^2.8.6"
-GDAL = "*"
+GDAL = "^3.2.1"
```

Comment:
why are we setting explicit version here?

Reply:
I thought we needed to for MakeValid to work.

Reply:
we don't know when makevalid was incorporated to GDAL

Reply:
but yeah, it's probably for the best